src/app/components/PageLayoutWrapper/index.test.tsx

```diff
@@ -7,7 +8,7 @@ global.performance.getEntriesByName = jest.fn(() => []);
 describe('PageLayoutWrapper', () => {
   it('should render default page wrapper with children', async () => {
     const { container } = render(
-      <PageLayoutWrapper pageData={{}} status={200}>
+      <PageLayoutWrapper pageData={{ metadata: { type: 'home' } }} status={200}>
```

Comment:
Would this work (so that it's really generic)

```suggestion
      <PageLayoutWrapper pageData={{ metadata: { type: 'test-page-type' } }} status={200}>
```